compiler/compiler.go

```diff
@@ -1571,8 +1571,9 @@ func (b *builder) createExpr(expr ssa.Value) (llvm.Value, error) {
 	switch expr := expr.(type) {
 	case *ssa.Alloc:
 		typ := b.getLLVMType(expr.Type().Underlying().(*types.Pointer).Elem())
-		if expr.Heap {
-			size := b.targetData.TypeAllocSize(typ)
+		size := b.targetData.TypeAllocSize(typ)
+		// move all allocations larger than 256 bytes to the heap
```

Comment:
Actually, can you maybe add a comment that this matches `maxStackAllocs` in transform/allocs.go? And maybe also a comment to transform/allocs.go to point back here? Just so that anybody looking at the code in the future knows that both values need to be updated at the same time.

Reply:
Comments added.